aep/general/0202/aep.md.j2

```diff
@@ -0,0 +1,52 @@
+# Fields
+
+The [`buf.validate.field`][protovalidate] type, through its accompanying
+extension `buf.validate.field`, enriches a field's schema beyond the basic
+name and type information.
+
+## Guidance
+
+Decorating a field with `buf.validate.field` is only necessary when a field
+consistently follows a set pattern.
```

Comment:
```suggestion
APIs **should** use `buf.validate.field` to annotate fields with constraints and validation rules.
```

I think guidance needs to have a **may**, **should**, **should not**, **must**, or **must not** somewhere - no?